lib/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Disallows keywords in your comments, such as TODO or FIXME
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * `"disallowKeywordsInComments": true`
+ *
+ * Invalid:
+ * ```
+ * // ToDo
+ * //TODO
+ * /** fixme *\/
+ * /**
+ *  * FIXME
+ *  *\/
+ * ```
+ */
+
+var assert = require('assert');
+
+var r = /\b(todo|fixme)\b/i;
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(disallowKeywordsInComments) {
+        assert(
+            disallowKeywordsInComments === true,
+            'disallowKeywordsInComments option disallows certain keywords in your comments'
+        );
+    },
+
+    getOptionName: function() {
+        return 'disallowKeywordsInComments';
+    },
+
+    check: function(file, errors) {
+        file.getComments().forEach(function(comment) {
+            var errorLoc = {};
+            // split by new line to get accurate line number reporting
+            var splitComment = comment.value.split('\n');
+
+            splitComment.forEach(function(commentNode, index) {
+                var matchIndex = commentNode.search(r);
+
+                if (matchIndex >= 0) {
+                    // offset reporter column indicator
+                    // because parser strips the chars that begin comments (//, /*)
+                    if (index === 0) {
+                        matchIndex += 2;
+                    }
+
+                    // adjust line location for reporter
+                    // comment is split by new line and iterated over
+                    errorLoc.line = comment.loc.start.line + index;
```

Comment:
Inline this object directly into the error.add call. No need to define as empty object first, nor even assign to a var.